packages/storefront-lwc/scripts/runtime.mjs

```diff
@@ -57,18 +81,17 @@ const mode = process.env.NODE_ENV || 'development';
 
     // start the server
     const server = sampleApp.expressApplication.listen(port, () => {
+        const portToTellUser =
+            process.env.SFCC_DEV_MODE === 'true' ? 3000 : server.address().port;
```

Comment:
Tell the user the port the app is on based upon dev/prod mode. Dev mode it is always 3000, while prod is according to the env variable.